test/mocks/Test.js

```diff
@@ -23,4 +23,14 @@ describe('Test', function() {
       test.slow().should.equal(200);
     });
   });
+
+  describe('#withEmptyFile', function() {
+    it('should return the test and set the file to undefined', function() {
+      // when
+      var test = new Test().withUndefinedFile();
+
+      // then
+      should.equal(test.file, undefined);
```

Comment:
I tend to prefer the use of expect in cases like this, it's more in keeping with the grammatical style of this kind of assertion framework. ie. `expect(test.file).to.be.undefined`. Not a big deal though, merging anyway.